packages/@ourworldindata/grapher/src/lineCharts/LineChart.tsx

```diff
@@ -1285,6 +1316,7 @@ export class LineChart
             points,
             seriesName,
             isProjection: column.isProjection,
+            isConnected: !column.display?.plotMarkersOnlyInLineChart,
```

Comment:
Up to you, but maybe it makes sense to rename this from `isConnected` to `plotMarkersOnly`, to have consistent naming across the board?

Reply:
Yeah, that would be nicer. It's left over from first having named it `isConnected`. I'll rename!